tensorhive/config.py

```diff
@@ -29,6 +29,19 @@ class APIConfig():
     TITLE = 'TensorHive API'
     VERSION = '1.0'
 
+    JWT = {}
```

Comment:
This will definetely change after merge with my patch, but that's totally fine cause I'll handle this by myself.

Reply:
Refactored.